charts/argo-cd/values.yaml

```diff
@@ -1,76 +1,459 @@
-## CoreWeave Custom Options
-region: ORD1
-
-affinity: {}
-tolerations: []
-
-# -- Provide a custom dns name in place of coreweave's ingress. Requires custom Ingress Controller
-customExternalDnsName: ''
-# -- Provide the annotation value for your ingress controller you wish to use.
-customIngressControllerClassAnnotation: ''
-
-## Argo CD configuration
-## Ref: https://github.com/argoproj/argo-cd
-##
-# -- Provide a name in place of `argocd`
-nameOverride: argocd
-# -- String to fully override `"argo-cd.fullname"`
-fullnameOverride: ""
-# -- Override the Kubernetes version, which is used to evaluate certain manifests
-kubeVersionOverride: ""
-
+## Globally shared configuration
 global:
-  skipCrds: true 
-
+  # -- Common labels for the all resources
+  additionalLabels: {}
+    # app: argo-cd 
+  cluster: 
+    ngcp: true
```

Comment:
can we default this to false? and keep region ORD1 for standard non NGCP installs.

Reply:
yup that's an error